build.gradle

```diff
@@ -535,6 +535,7 @@ jsonSchema2Pojo {
 sonarqube {
     properties {
         property "sonar.exclusions", "build/generated-sources/**/*.java,**/AuditRepository.java"
+        property "sonar.cpd.exclusions", "**/data/casedetails/*.java,**/model/std*.java"
         property "sonar.projectName", "ccd-data-store-api"
```

Comment:
property "sonar.cpd.exclusions", "**/data/casedetails/*.java,**/model/std*.java". Why we are excluding classes under /src/main/java/uk/gov/hmcts/ccd/domain/model/std/

Reply:
removed this now